Create landing page

FabianAronsson opened the pull request:
Adds design to the landing page. 

Perhaps the main logo should be white instead? 
And also, any thoughts on the gradient? Does it fit?

Closes #83

Mojtabarezaei4 reviewed (commented):
![image](https://github.com/Mojtabarezaei4/Queueomatic/assets/54796834/64838f0c-1dea-4867-b6e8-a38b70445ef9)

Lovely design.

* I think the main logo seems nice as black but maybe a little smaller?
* The logo on the cart at the bottom, could be in white color without a background.
* Could you apply the background to the app.css? Like for the whole HTML element? I think it should solve the problem plus we can have it as universal.

FabianAronsson commented:
Updated so the gradient is displayed at all times. Also changed the logo on the card, but I am not sure if it worked correctly. You can see some small artifacts from it.

Mojtabarezaei4 reviewed (approved):
Great job.
